.buildkite/steps/extract-agent-version-metadata.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -euo pipefail
 
-agent_version=$(awk -F\" '/var baseVersion string = "/ {print $2}' agent/version.go)
+agent_version=$(<"agent/VERSION")
```

Comment:
my bash-fu is leaving me: is this the same as having 
```
agent_version=$(cat agent/VERSION)
```

if so, can we use cat rather than the hard-to-google bash syntax?

Reply:
The term-of-art is [input redirection](https://www.gnu.org/software/bash/manual/html_node/Redirections.html#Redirecting-Input). Shell purists would [throw a fit](https://en.wikipedia.org/wiki/Cat_(Unix)#Useless_use_of_cat), but cat is more obvious?

Reply:
readability > purism, imo. shell purists can stay mad

Reply:
Rebased, and added `xargs` to trim newlines